### PR TITLE
Create plugin-splunk-for-jenkins

### PR DESCRIPTION
**Pull request by fengxx**
Request publish permission for artifact id
"com.splunk.splunkins.pom"
"com.splunk.splunkins.splunk-devops"
"com.splunk.splunkins.splunk-devops-extend"

### CODE REVIEW

**Comment by orrc** [comment]
https://issues.jenkins-ci.org/browse/HOSTING-201
https://github.com/splunk/splunkforjenkins/

**Comment by orrc** [comment]
From your hosting request, it seems like you want to host two plugins, plus a parent POM.

AFAIK, this requires three new files in this repo; each with the `name` value matching the POM `artifactId`, but I'll leave it to @daniel-beck to confirm.

**Comment by fengxx** [comment]
Thanks for reviewing, I splitted into 3 files

**Comment by daniel-beck** [comment]
Yep. For now it needs to be separate files. Sorry about that.

**Comment by orrc** [comment]
Hosted: https://github.com/jenkinsci/splunk-devops-plugin

@daniel-beck: Can you check the separate files look good, please?

File: permissions/plugin-splunk-devops-extend.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+name: "splunk-devops-extend"
+paths:
+- "com/splunk/splunkins/splunk-devops-extend"
+developers:
+- "fengxx"
+- "araza_splunk"
```
Comment (daniel-beck) [inline permissions/plugin-splunk-devops-extend.yml]: `araza_splunk` does not existing in Artifactory (see this repo's README) so will prevent these permissions to be applied for everyone.

**Comment by fengxx** [comment]
araza_splunk logged in repo, issue fixed.

**Pull request merged**